drivers/SmartThings/sonos/src/api/sonos_connection.lua

```diff
@@ -92,15 +92,28 @@ local function _open_coordinator_socket(sonos_conn, household_id, self_player_id
   end
 
   if coordinator_id ~= self_player_id then
-    local coordinator = sonos_conn.driver.sonos:get_household(household_id).players[coordinator_id]
-    _, err = Router.open_socket_for_player(coordinator_id, coordinator.websocketUrl)
+    local household = self.driver.sonos:get_household(household_id)
```

Comment:
`self` isn't available here to access the `sonos` instance, `sonos` is available on `sonos_conn.driver` though.